### PR TITLE
Add SIG Architecture Code Organization spotlight article

### PR DESCRIPTION
**Pull request by fsmunoz**
SIG Architecture: Code Organization spotlight article.

Initial reviews will be done at the `contributor-site` PR https://github.com/kubernetes/contributor-site/pull/493 ; after they are integrated, I will sync the content of this PR and remove the WIP / unhold.

### CODE REVIEW

**Comment by sftim** [comment]
/hold

pending assignment of publication date

**Comment by fsmunoz** [comment]
/retitle Add SIG Architecture Code Organization spotlight article

**Comment by fsmunoz** [comment]
Synced content with the reviewed version at https://github.com/kubernetes/contributor-site/pull/493 , including publication date.

File: content/en/blog/_posts/2024-04-11-SIG-Architecture-Code-Organization-Spotlight.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Code Organization"
+slug: sig-architecture-code-spotlight-2024
+canonicalUrl: https://www.kubernetes.dev/blog/2024/04/11/sig-architecture-code-spotlight-2024
+date: 2024-04-11
+---
+
+** Author: Frederico Muñoz (SAS Institute)**
```
Comment (sftim) [inline content/en/blog/_posts/2024-04-11-SIG-Architecture-Code-Organization-Spotlight.md:9]:
```suggestion
**Author: Frederico Muñoz (SAS Institute)**
```

**Pull request merged**